src/Nethermind/Nethermind.Synchronization/Peers/SyncPeersReport.cs

```diff
@@ -62,7 +62,8 @@ public void WriteFullReport()
                 }
 
                 RememberState(out bool _);
-                _stringBuilder.Append($"Sync peers - Initialized: {_currentInitializedPeerCount} | All: {_peerPool.PeerCount} | Max: {_peerPool.PeerMaxCount}");
+                _stringBuilder.AppendLine($"Sync peers - Initialized: {_currentInitializedPeerCount} | All: {_peerPool.PeerCount} | Max: {_peerPool.PeerMaxCount}");
+                AddPeerHeader();
```

Comment:
We sometimes have headers descriptions without any data below. Maybe let's print headers only if OrderedPeers is not empty?

<img width="659" alt="header" src="https://user-images.githubusercontent.com/77129288/184329685-1ed209c0-0774-4cfe-8b3b-a8984d52388c.PNG">

Reply:
@smartprogrammer93